src/sentry/search/snuba/backend.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+
+from sentry.search.django.backend import (
+    DjangoSearchBackend, QuerySetBuilder, CallbackCondition, ScalarCondition,
+    get_sort_clause, sort_strategies
+)
+from sentry import tagstore
+
+
+class SnubaSearchBackend(DjangoSearchBackend):
+    def _query(self, project, start, end, retention_window_start, group_queryset, tags=None,
+               environment=None, sort_by='date', limit=100, cursor=None, count_hits=False,
+               paginator_options=None, **parameters):
```

Comment:
Would it be worth asserting that `tagstore` is the expected implementation here?

Reply:
I don't follow. :(